src/scss/settings/_variables.scss

```diff
@@ -104,9 +104,7 @@ $spacing: 8px; // only for use in map below – not for in components
 // spacing map
 // Can access levels by using the spacing function > spacing() or spacing(x5) for example
 $spacing: (
-    x0.5 : $spacing / 2,
```

Comment:
Out of interest, what's the character that makes this invalid?

I think when we come to add the design tokens for spacing from the PIE design tokens, this issue will disappear as they'll be set there instead.

Could we possibly hold off on this change until after the colour updates (we'll be putting them in this week/early next week). Then we could potentially just update the design tokens to have the spacing values and replace this whole spacing map entirely? ([here are the tokens that it'll be changing too](https://docs.google.com/spreadsheets/d/1KRVJgMELOA-QkKuqNZh-9z4kpQe__diuRhjIRLxNR1Q/edit#gid=1188492957)).

Reply:
It's the dot in the key name that is invalid.

Reply:
Sure, can hold off. I was just looking at the issues in using Vite to compile our Vue packages again and this is the first thing that is flagged up when using Vite with dart-scss.

Reply:
Makes sense. I wonder if it would make it valid to change it to be `'x0.5' : $spacing / 2,` potentially?

May fix the parsing issue then if that's valid syntax.

Reply:
Ah, ok. That makes sense. I'll try and have a look at this and replace these spacing variables with the design tokens, as they should have new names which will solve this issue (and save us having to make 2 similar breaking changes then).